src/openapi/openapi.json

```diff
@@ -6128,6 +6128,134 @@
         "x-router-controller": "cors"
       }
     },
+    "/api/data/workflowcreate/step": {
```

Comment:
I think this can probably just be `"/api/data/workflow/step"` with the implication that a post to this step would create a step

Reply:
`/api/data/workflow/step` not used because there is another `/api/data/workflow/{id}` endpoint which is capturing the new call and assuming `step` is the workflow id. I'll need to look more into whether we want to adapt a new format for the existing endpoint or new endpoint and get back to this comment.